text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,343 @@
+ 
+Options:
+    --set-recommended             set feature flags to their recommended states 
+      --all                          set all feature flags to their recommended states     
+      --unconfigured                 set unconfigured feature flags to their recommended states     
+    --set-default                 set feature flags to their default states
+      --all                          set all feature flags to their default states
+      --unconfigured                 set unconfigured feature flags to their default states
+    "#FLAGNAME#"                  modify a specific flag's state
+      --set-recommended              set specific feature flag to its recommended state
+      --set-default                  set specific feature flag to its default state
+      --set                          set specific feature flag to any state
+    
+Examples:
+    > cdk flags --set-recommended --all
+        Feature Flag                              Recommended Value            User Value
+      * @aws-cdk/...                              true                         false
+      * @aws-cdk/...                              true                         false
+      * @aws-cdk/...                              true                         - <unset>
+      Setting all feature flags to the recommended values...
+
+    > cdk flags --set-default --unconfigured
+        Feature Flag                              Recommended Value            User Value
+      * @aws-cdk/...                              true                         - <unset>
+      Setting your unconfigured flags to the default values...
+
+    > cdk flags "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021" --set
+      What value would you like to change "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021" to?
+      > true
+        false
+```
+
+## README
+
+The new CLI command `cdk flags` shows you a report of your feature flag configurations that differ from our recommended states. It
+compares your current feature flag configurations and our recommended states. When you run `cdk flags`, a feature flags report is displayed
+to you in the following format: flags that do not match the recommended values and then flags that are not configured by the user at all.
+There will be a Menu with options for the user to configure their feature flags.
+
+```
+> cdk flags 
+    Feature Flag                              Recommended Value            User Value
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         - <unset>
+  
+  How would you like to proceed?
+   >  Set all flags to recommended values 
+      Set unconfigured flags to recommended values
+      Reset unconfigured flags to defaults (no impact)
```

Comment:
"defaults" is good, but it is also confusing. New default or old default? 

```suggestion
      Record current behavior as intentional for all unconfigured flags (no impact)
```

Or something like that... I'm having trouble coming up with better wording.